imports/plugins/core/accounts/server/methods/userAccountQuery.js

```diff
@@ -0,0 +1,29 @@
+import { Meteor } from "meteor/meteor";
+import { Accounts } from "/lib/collections";
+import { Reaction } from "/lib/api";
+
+/**
+ * @name userAccountQuery
+ * @method
+ * @summary query the Accounts collection and return user account data
+ * @param {Object} context - an object containing the per-request state
+ * @param {String} id - id of user to query
+ * @return {Object} user account object
+ */
+export function userAccountQuery(context, id) {
+  const userId = context.user._id;
```

Comment:
I'd suggest passing only `userId` since this is unlikely to ever need anything else that's on `context`.

Also, `context.user` will be `null` for an anonymous request, so you'll have to guard against that. But the `user` prop will always be there. I suggest:

```js
const { _id: userId } = context.user || {};
```

Reply:
I see what you're saying, it makes sense to only pass the one `id` in. My thought here was trying to standardize passing the parameters in, always having `context` as the first parameter.

In this case, we only do need the `id`, but in other instances we will need need more from the `context`.

Reply:
Yes that could be true. Whatever you want to do is fine. It will become clear eventually what the best pattern is.